.github/workflows/unpinned.yml

```diff
@@ -0,0 +1,43 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
+
+name: Unpinned tests at latest release
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 2 * * 1-5'
+
+jobs:
+  setup:
+    name: Setup variables
+    runs-on: 'ubuntu-20.04'
```

Comment:
Since this is a test to determine if things works for a typical user perhaps it makes sense to run this on other platforms, Windows and Mac also.

But I don't know if we have any system wide dependencies.
If everything is expected to work the same on all systems then it's probably not necessary.

Reply:
Yes, that may make sense eventually. But I think it is less likely to happen than, e.g., problems from different Python versions, so we should first consider extending that matrix.